src/libsyncengine/reconciliation/platform_inconsistency_checker/platforminconsistencycheckerworker.cpp

```diff
@@ -36,8 +36,24 @@ void PlatformInconsistencyCheckerWorker::execute() {
 
     _idsToBeRemoved.clear();
 
-    ExitCode exitCode = checkTree(_syncPal->updateTree(ReplicaSide::Remote)->rootNode(),
-                                  _syncPal->updateTree(ReplicaSide::Remote)->rootNode()->name());
+    ExitCode exitCodeRemote = checkRemoteTree(_syncPal->updateTree(ReplicaSide::Remote)->rootNode(),
```

Comment:
**Suggestion.** 
Define a small helper method `ExitCode checkTree(const ReplicaSide side)` that calls `checkRemoteTree` and `checkLocalTree` depending on its argument.